<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebSocketClient</class>
 <widget class="QWidget" name="WebSocketClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>38</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recv:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Send:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editSend">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>651</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;I'm Client.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>风速控制</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>347</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Url</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editUrl">
      <property name="text">
       <string>ws://localhost:12345</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>155</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="footLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>模式选择</string>
   </property>
   <widget class="QRadioButton" name="makecold">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>制冷</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="makehot">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>制热</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>温度控制</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="opening">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>开机</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="editRecv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="RealTemp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="RealWind">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>空调温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>空调风速</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="RoomTmp">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前室内环境温度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
